README.md

```diff
@@ -322,7 +322,47 @@ When a request does not require a server-side render, `ctx.body` follows regular
 In the server, `ctx` also exposes the same properties as a [Koa context](http://koajs.com/#context)
 
 * `ctx: Object`
-  * `header: Object` - alias of `ctx.headers`
+  * `req: http.IncomingMessage` - [Node's `request` object](https://nodejs.org/api/http.html#http_class_http_incomingmessage)
+  * `res: Response` - [Node's `response` object](https://nodejs.org/api/http.html#http_class_http_serverresponse)
+  * `request: Request` - [Koa's `request` object](https://koajs.com/#request)
+  * `response: Response` - [Koa's `response` object](https://koajs.com/#response)
+  * `state: Object` - recommended namespace for passing information through middleware and to your frontend views `ctx.state.user = await User.find(id)`
+  * `throw: (status: ?number, message: ?string, properties: ?Object) => void` - throws an error
+    * `status: number` - HTTP status code
+    * `message: string` - error message
+    * `properties: Object` - is merged to the error object
+  * `assert: (value: any, status: ?number, message: ?string, properties: ?Object)` - throws if `value` is falsy. Uses [Assert](https://github.com/jshttp/http-assert)
+    * `value: any`
+    * `status: number` - HTTP status code
+    * `message: string` - error message
+    * `properties: Object` - is merged to the error object
+  * `respond: boolean` - set to true to bypass Koa's built-in response handling. You should not use this flag.
+  * `app: Object` - a reference to the Koa instance
+  * `cookies: {get, set}` - cookies based on [Cookie Module](https://github.com/pillarjs/cookies)
+  
+  <details><summary>View Koa cookies details</summary>
```

Comment:
I still think we want something that signifies that we access these properties through the `cookies/response/request` keys.  Does something like this work?

```
* `response: Object`: <details>...
```